src/System.Runtime/tests/System/TupleTests.cs

```diff
@@ -327,66 +354,74 @@ public static void ToStringTest()
             tupleDriverA.ToString("(" + ((short)10000) + ", " + ((int)1000000) + ", " + ((long)10000000) + ", 2008?7?2?, 0, " + ((Single)0.0001) + ", " + ((Double)0.0000001) + ", " + now + ", (False, System.Object), " + TimeSpan.Zero + ")");
         }
 
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, ".NET Framework hasn't received the fix for https://github.com/dotnet/corefx/issues/18528 yet.")]
+        public static void Equals_GetHashCode() => Equals_GetHashCode(false);
+
         [Fact]
-        public static void Equals_GetHashCode()
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework, ".NET Framework hasn't received the fix for https://github.com/dotnet/corefx/issues/18528 yet.")]
+        public static void Equals_GetHashCode_netfx() => Equals_GetHashCode(true);
```

Comment:
Sure thing. Debated that because it's literally the only purpose of the entire couple methods.